guide/design/motion.md

```diff
@@ -1,13 +1,200 @@
-# Motion
+<h1> Motion principles <span style="display:inline; font-size: 16px; font-weight:bold;">(BETA)</span> </h1>
```

Comment:
plz convert to Markdown

Reply:
Couldn't get the `(BETA)` part to look acceptable without custom HTML. 
Is there a quick Markdown solution for this?
Going by the rest of the guide, I figured HTML was deemed acceptable when necessary.